modules/snooker/src/main/ts/ephox/snooker/api/ResizeBehaviour.ts

```diff
@@ -75,12 +79,32 @@ const resizeTable = (): ResizeBehaviour => {
     }
   };
 
+  const getNewWidths = (_table: SugarElement<HTMLTableElement>, warehouse: Warehouse, tableSize: TableSize, pixelDelta: number) => {
+    const sizes = tableSize.getWidths(warehouse, tableSize);
+
+    if (tableSize.isRelative) {
+      const tableWidth = tableSize.pixelWidth() + pixelDelta;
+      const ratio = tableWidth / tableSize.pixelWidth();
```

Comment:
To save getting the width twice from the DOM, maybe something like this?
```suggestion
      const tableWidth = tableSize.pixelWidth();
      const ratio = tableWidth + pixelDelta / tableWidth;
```